docs/service_notifications.md

```diff
@@ -0,0 +1,168 @@
+# Firefox Accounts Attached Service Notifications.
+
+Mozilla-hosted services that integrate with Firefox Accounts
+have certain additional data-handling requirements,
+such as a requirement to delete user data
+when a user deletes their Firefox Account.
+
+To help services meet these requirements,
+FxA publishes a stream of SQS notification events.
+If you would like to connect your service
+as a consumer of this stream,
+please email fxa-staff@mozilla.com for assistance.
+
+Each notification event has a stringified JSON body
+with an "event" key naming the type of event.
+The types of event, the additional data they contain,
+and the actions that attached services should take in response
+are as follows.
+
```

Comment:
I think it might be worthwhile to add here that all messages have a `ts` (seconds float) and can optional have a `metricsContext` payload.